docs/deploy-development-rp.md

```diff
@@ -242,6 +242,12 @@ After that, when you [create](https://github.com/Azure/ARO-RP/blob/master/docs/d
   curl -X GET -k "https://localhost:8443/admin/subscriptions/$AZURE_SUBSCRIPTION_ID/resourceGroups/$RESOURCEGROUP/providers/Microsoft.RedHatOpenShift/openShiftClusters/$CLUSTER/kubernetespodlogs?podname=$POD&namespace=$NAMESPACE&container=$CONTAINER"
   ```
 
+* List Supported VM Sizes
+  ```bash
+  VMROLE=<master or worker>
+  curl -X GET -k "https://localhost:8443/admin/listSupportedVMSizes?instanceType=$VMROLE"
```

Comment:
`instanceType` should be `vmRole`

Reply:
Yes, missed this, made the change. thanks

Reply:
And the URI name change actually makes sense, modified to "supportedvmsizes"